README.md

```diff
@@ -2,32 +2,110 @@
 
 [![Latest Version on Packagist](https://img.shields.io/packagist/v/cybex/laravel-protector.svg?style=flat-square)](https://packagist.org/packages/cybex/laravel-protector)
 
-Protect Databases by generating Backups and Import these on non-productive Environments.
-Authenticate users with the usage of Laravel Sanctum Tokens.
-Encrypt your database dumps with Sodium Encryption.
+Allows a client to download and import a live server database backup.
```

Comment:
Description is lacking a lot of available features.

* Encrypted or plain backups
* Generation of local backups
* Import of existing backups
* Downloading backups from a remote source with or without import